packages/std/src/mock.rs

```diff
@@ -406,6 +411,23 @@ mod test {
         assert_eq!(a, c);
     }
 
+    #[test]
+    fn mock_env_info_arguments() {
+        let name = HumanAddr("my name".to_string());
+
+        // make sure we can generate with &str, &HumanAddr, and HumanAddr
+        let (env1, info1) = mock_env_info("my name", &coins(100, "atom"));
+        let (env2, info2) = mock_env_info(&name, &coins(100, "atom"));
+        let (env3, info3)= mock_env_info(name, &coins(100, "atom"));
```

Comment:
Space missing here. Do you have a rustfmt plugin for your IDE? I have one on VSCode and it formats automatically on save.